frontend/src/views/TE/Task.vue

```diff
@@ -174,6 +174,12 @@ import TaskMixin from "@/components/TE/TaskMixin";
 import FormatTime from "@/mixins/FormatTime";
 import VersionDownloadLink from "@/components/TE/VersionDownloadLink.vue";
 
+enum TaskState {
```

Comment:
I'd say it's not a task state but the page state. Did other pages there will be also state enum, and no need to prefix it with a specific term. There will be usually only one enum per page.